package.json

```diff
@@ -9,6 +9,11 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.1",
+    "@types/jest": "^26.0.20",
```

Comment:
these are just dependencies that you need for typescript.

you can optionally use it if you want to play around (you can still use .js files no sweat)